### DISCUSSION TITLE
Can't find platform url field

### DISCUSSION THREAD

**Discussion by manojahi** [Budibase/budibase#3530] [Help]
Hi there,

I am trying to use Google oauth and found that i may need to add platform url so that my google redirect url can be a full url.
but i am not seeing any field for platform url in `Organisation` page.

I am using budibase cloud for hosting. I think that might be the reason i am not able to find the platform url field?
If so, how can i use google redirection url because google is throwing error saying redirection url mismatch as its redirecting to `http` not `https`

thanks in advance

**Comment by mjashanks** [comment]
your platform url will be something like `https://mycustomdomain.budibase.app`

**Reply by manojahi** [reply]
Thanks, but this URL is not working for google login redirection. google is redirecting to `http://...` but my platform url is `https://`
any thoughts?
